README.md

```diff
@@ -133,17 +196,31 @@ To avoid accidentally exceeding the rate limit on [Google's Places API][google-p
   ```
 [google-places]: https://developers.google.com/places/javascript/
 
-## Quick Start 
+#### 3. Run the App
+To run the application use:
+```bash
+rails server
+```
+and
+```bash
+rails console
+```
+If you setup any optional dependencies (Mailtrap, Google Places API), 
+preappend project commands with `foreman run`:
+```bash
+foreman rails server
+```
+and
+```bash
+foreman rails console
+```
```

Comment:
Maybe change the sequence: First the setup _with_ dependencies, then the setup without. ??